ESP8266WirelessPrintAsync/ESP8266WirelessPrintAsync.ino

```diff
@@ -714,7 +715,8 @@ void loop() {
   }
 
   SendCommands();
-  ReceiveResponses();
+  if (!commandQueue.isAckEmpty())
```

Comment:
This breaks autobaud detection, it should always be listening for messages

Reply:
That's weird too... when M115 is inserted to queue test should let it be called...
I saw a flaw in serial receive code, I'll try to change a little bit how it works. Bad thing is that I can't connect my printer yet and I'm blinded

Reply:
But for sure will break autotemp frame read